blocks/feed/feed.js

```diff
@@ -139,7 +152,17 @@ export default async function decorate(block) {
     .take(blockCfg.count ? parseInt(blockCfg.count, 10) : 4)
     .toList();
   block.innerHTML = '';
-  const blockContents = resultParsers[blockType](results, blockCfg);
+  const meta = getMetadata('page-style');
+  if (meta === 'featured' && blockType === 'highlight') {
+    let locale = (getLanguage(
+      window.location.pathname,
+      false,
+    ));
+    if (locale !== 'en') locale = 'jp';
```

Comment:
Why do we need this? If the default locale is to be `jp` , it should be handled in `getLanguage`

Reply:
Yes I was getting ja as the output of getLanguage, that's why. Seems like Today Ashutosh has done some changes to make the getLanguage - - -> resulting in "ja". May be once merged to main, will remove this check

<img width="1375" alt="image" src="https://github.com/hlxsites/sunstar-foundation/assets/37831850/ff792fad-dbf1-436b-9084-bc1e03f3a1d0">